src/scss/includes/z_index.scss

```diff
@@ -35,7 +35,7 @@
 }
 
 .top_bar {
-  z-index: 3;
+  z-index: 2;
```

Comment:
why this change ? 
As a result, this top bar is no longer visible on mobile when detailed POI panel is open:
![image](https://user-images.githubusercontent.com/4726554/72539921-61713580-3880-11ea-89de-61a8dd09153c.png)

Reply:
This was apparently the result of an outdated base branch, can't reproduce since the last rebase.